decavision/model_testing/testing.py

```diff
@@ -231,3 +231,21 @@ def evaluate(self, path):
         generator = self._load_dataset(path)
         results = self.model.evaluate(generator)
         print('Accuracy of', results[1] * 100, '%')
+    
+    def generate_classification_report(self, path):
+        """
+        Compute classification report resulting from predictions on a dataset of images.
```

Comment:
Can you write the main quantities tha are displayed?

Reply:
Also specify that the method prints the results.

Reply:
Done!